### PR TITLE
Add news entry about GSoC 2017.

### PR DESCRIPTION
**Pull request by pulkomandy**

### CODE REVIEW

File: content/news/2017-02-27_back_into_google_summer_of_code.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+<p><a href="/community/gsoc/2017">Read more</a> about Google Summer of Code 2017</p>
+<p><a href="/community/gsoc/2017/ideas">List of ideas</a> the student will build their applications from</p>
+
+
+<p>If you know people in the right age bracket that might be interested, point them
+to <a href="https://summerofcode.withgoogle.com">Google Summer of Code site</a> that has all the information.</p>
+
+<p>Students must submit their application after March 20, but shouldn't wait until
+then to start preparing it. The coding period spans over june and july, and august.</p>
```
Comment (kainjow) [inline content/news/2017-02-27_back_into_google_summer_of_code.md]: Months should be capitalized?

File: content/news/2017-02-27_back_into_google_summer_of_code.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+and related applications, with help from our mentor team. Google gives the
+students a stipend, which allows them to work full-time on the project without
+the need to find another job.</p>
+
+<p>The goal of the program is to introduce students to the way open source
+project works, and encourage them to become long term contributors. Several
+members of the Haiku developer team are indeed former GSoC students.</p>
+
+<p><a href="/community/gsoc/2017">Read more</a> about Google Summer of Code 2017</p>
+<p><a href="/community/gsoc/2017/ideas">List of ideas</a> the student will build their applications from</p>
```
Comment (waddlesplash) [inline content/news/2017-02-27_back_into_google_summer_of_code.md]: student -> students

File: content/news/2017-02-27_back_into_google_summer_of_code.md (hunk 1 above)
Comment (waddlesplash) [inline content/news/2017-02-27_back_into_google_summer_of_code.md]: Indeed they should (@PulkoMandy showing he is a native French speaker here. :)

Comment (kainjow) [inline content/news/2017-02-27_back_into_google_summer_of_code.md]: It should also be "June, July and August". No need for multiple ands.

**Comment by waddlesplash** [comment]
For some reason, this didn't close when the (manual) merge occurred...

**Pull request closed without merging**